client/components/jetpack/card/jetpack-product-card/display-price/time-frame.tsx

```diff
@@ -184,6 +208,15 @@ const TimeFrame: React.FC< TimeFrameProps & A11yProps > = ( {
 	// `1 === discountedPriceDuration` condition taken from client/my-sites/plans/jetpack-plans/product-lightbox/payment-plan.tsx:56
 	if ( isDiscounted ) {
 		if ( 1 === discountedPriceDuration && formattedOriginalPrice ) {
```

Comment:
Feel free to ignore: Shall we add a comment in code saying this means a monthly free trial on the yearly plan?